src/components/Dialogs/LanguagesModal.tsx

```diff
@@ -10,7 +10,7 @@ import { useGetSettingsQuery, usePatchSettingsMutation } from '@/core/rtkQuery/s
 export const languageDescription = {
   'x-jat': 'Romaji (x-jat)',
   en: 'English (en)',
```

Comment:
@krbrs place `Pinyin (x-zht)` above `English (en)`. I'll add it to the default list on the server side too, since donghua have `x-zht` for their main title and @harshithmohan wanted it to be in the default list for it to be placed near `Romaji (x-jat)`.